src/data/thunks.js

```diff
@@ -235,8 +292,8 @@ export function pollAttempt(url) {
       const data = await pollExamAttempt(url);
       const updatedAttempt = {
         ...currentAttempt,
-        time_remaining_seconds: data.time_remaining_seconds,
-        accessibility_time_string: data.accessibility_time_string,
+        timeRemainingSeconds: data.timeRemainingSeconds,
```

Comment:
I think this should still be in snake case, because that's the field being returned by the backend